pyproject.toml

```diff
@@ -19,13 +19,14 @@ dependencies = [
     "networkx>=2,<4",
     "pydantic",
     "PyYaml>=6",
-    "typing_extensions>=4.4.0"
+    "typing_extensions>=4.4.0",
+    "packaging",
```

Comment:
my bad: 

packaging is used here: https://github.com/mindee/tawazi/blob/e7882c29288ee504d0d5f1dc188485aa777e7ba0/tawazi/config.py#L4